### PR TITLE
added front end

### PR DESCRIPTION
**Pull request by ziqiang2344**

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -4,26 +4,63 @@
 <head>
   <meta charset="utf-8">
   <title>PDF to Text Converter</title>
-  <!-- <link rel="stylesheet" href="css/styles.css"> -->
   <link rel='stylesheet' type='text/css' href='css/style.php' />
+  <div class="header">
```
Comment (Michelle-Lohwt) [inline index.php:8]: @ziqiang2344 Boss, can you make the indentation nice a bit? (very messy, don't know which tag is for which tag)

File: css/style.php [hunk 2]
```diff
@@ -2,27 +2,161 @@
...
+  font-size: 20px;
+  font-family:
+}
+
+@keyframes popout {
+    from{transform:scale(0)}
+    80%{transform:scale(1.1)}
+    to{transform:scale(1)}
+}
+
```
Comment (Michelle-Lohwt) [inline css/style.php:162]: @ziqiang2344 Boss, can you make the indentation nice a bit?

File: css/style.php [hunk 3]
```diff
@@ -2,27 +2,161 @@
...
+    font-size: 20px;
+    font-family: Arial, Helvetica, sans-serif;
+    top:50%;
+    left: 50%;
+    color: white;
+}
+
+body,html{
+    height:100%;
+}
```
Comment (Michelle-Lohwt) [inline css/style.php:25]: why this one got `,` and what is body and html?

File: css/style.php [hunk 4]
```diff
@@ -2,27 +2,161 @@
...
+  right: 0;
+  top: 0;
+  bottom: 0;
+  z-index: -1;
+}
+
+@media (max-width: 750px) {
+    #background-video { display: none; }
+body {
+  /* The image used */
```
Comment (Michelle-Lohwt) [inline css/style.php:42]: Why body is in media?

File: css/style.php [hunk 5]
```diff
@@ -2,27 +2,161 @@
...
+  padding: 5px 20px;
+  background-color: black;
+  color: white;
+  text-align: center;   
+  font-size: 20px;
+  font-family:
+}
+
+@keyframes popout {
+    from{transform:scale(0)}
```
Comment (Michelle-Lohwt) [inline css/style.php:158]: What is this `@keyframes`?

File: css/style.php (hunk 3 above)
Comment (ziqiang2344) [reply css/style.php:25]: this selector is applying to full html code

File: css/style.php (hunk 4 above)
Comment (ziqiang2344) [reply css/style.php:42]: Not so sure why but the website's source code is writing this one

File: css/style.php (hunk 5 above)
Comment (ziqiang2344) [reply css/style.php:158]: This one is added by Derek @GitHubUser5136

File: index.php (hunk 1 above)
Comment (ziqiang2344) [reply index.php:8]: Okay sure no problem

File: css/style.php (hunk 2 above)
Comment (ziqiang2344) [reply css/style.php:162]: okay no problem

**Review by Michelle-Lohwt — APPROVED** [review]

**Pull request merged**